shell/src/main/java/org/apache/accumulo/shell/ShellOptionsJC.java

```diff
@@ -45,84 +40,12 @@
   @Parameter(names = {"-u", "--user"}, description = "username")
   private String username = null;
 
-  public static class PasswordConverter implements IStringConverter<String> {
-    public static final String STDIN = "stdin";
-
-    private enum KeyType {
-      PASS("pass:"), ENV("env:") {
-        @Override
-        String process(String value) {
-          return System.getenv(value);
-        }
-      },
-      FILE("file:") {
-        @SuppressFBWarnings(value = "PATH_TRAVERSAL_IN",
-            justification = "app is run in same security context as user providing the filename")
-        @Override
-        String process(String value) {
-          Scanner scanner = null;
-          try {
-            scanner = new Scanner(new File(value), UTF_8);
-            return scanner.nextLine();
-          } catch (IOException e) {
-            throw new ParameterException(e);
-          } finally {
-            if (scanner != null) {
-              scanner.close();
-            }
-          }
-        }
-      },
-      STDIN(PasswordConverter.STDIN) {
-        @Override
-        public boolean matches(String value) {
-          return prefix.equals(value);
-        }
-
-        @Override
-        public String convert(String value) {
-          // Will check for this later
-          return prefix;
-        }
-      };
-
-      String prefix;
-
-      private KeyType(String prefix) {
-        this.prefix = prefix;
-      }
-
-      public boolean matches(String value) {
-        return value.startsWith(prefix);
-      }
-
-      public String convert(String value) {
-        return process(value.substring(prefix.length()));
-      }
-
-      String process(String value) {
-        return value;
-      }
-    }
-
-    @Override
-    public String convert(String value) {
-      for (KeyType keyType : KeyType.values()) {
-        if (keyType.matches(value)) {
-          return keyType.convert(value);
-        }
-      }
-
-      return value;
-    }
-  }
-
   // Note: Don't use "password = true" because then it will prompt even if we have a token
   @Parameter(names = {"-p", "--password"},
       description = "password (can be specified as 'pass:<password>',"
```

Comment:
Should this document `stdin:` prefix and and a bare password with no prefix?

Reply:
I'm not sure why the original author choose not to describe these use-cases, so I left it as it was.

Reply:
I'm not sure either, but I wonder if it was because the shell already has a custom way of handling the stdin case if you don't use `-p` at all., so maybe it was a hidden, unneeded feature.

Reply:
I can try to fix it up the documentation when I merge this.